astacus/coordinator/plugins/clickhouse/steps.py

```diff
@@ -420,15 +420,17 @@ async def run_step(self, cluster: Cluster, context: StepsContext) -> None:
                 for client in self.clients
             ],
         )
+
+        def _escape_if_string(value):
```

Comment:
Missing parameter and return type hint.

Reply:
Extracting it and placing it below the imports